### DISCUSSION TITLE
get user by email address

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#73] [Q&A] [answered]
From [here](https://keygen.sh/docs/api/users/#users-list) I gather that it's possible to get "all" users, filtered by metadata. Unfortunately, this only refers to the extra metadata, not the the user's attributes. How to filter by email address? `metadata[email]=jane@example.com` doesn't work.

**Comment by ezekg** [comment, marked as answer]
You don't, since the `email` attribute is unique. Instead, you can request a user by their email:

```
GET /v1/accounts/<account>/users/jane@example.com
```
